stack.py

```diff
@@ -0,0 +1,21 @@
+from linkedlist import LinkedList
+
+
+class Stack:
+    def __init__(self, values=None):
+        self.ll = LinkedList(values)
```

Comment:
I'd call this self.storage, it happens to be a linked list.  Name things after what they are not what type they are.